x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/sync_jobs/documents_panel.tsx

```diff
@@ -45,7 +45,20 @@ export const SyncJobDocumentsPanel: React.FC<SyncJobDocumentsPanelProps> = (sync
       name: i18n.translate('xpack.enterpriseSearch.content.index.syncJobs.documents.volume', {
         defaultMessage: 'Volume',
       }),
-      render: (volume: number) => new ByteSizeValue(volume).toString(),
+      render: (volume: number) =>
+        volume < 1
+          ? i18n.translate(
+              'xpack.enterpriseSearch.content.index.syncJobs.documents.volume.lessThanOneMB',
+              {
+                defaultMessage: 'Less than 1mb',
+              }
+            )
+          : `${i18n.translate(
+              'xpack.enterpriseSearch.content.index.syncJobs.documents.volume.about',
+              {
+                defaultMessage: 'About',
+              }
+            )} ${new ByteSizeValue(volume * 1024 * 1024).toString()}`,
```

Comment:
To internationalize strings with values in them, the values need to be in the translation itself. You can't just concatenate, because word order differs between languages and translators cannot translate what's not in translation strings.

```suggestion
      render: (volume: number) =>
        volume < 1
          ? i18n.translate(
              'xpack.enterpriseSearch.content.index.syncJobs.documents.volume.lessThanOneMBLabel',
              {
                defaultMessage: 'Less than 1mb',
              }
            )
          : `${i18n.translate(
              'xpack.enterpriseSearch.content.index.syncJobs.documents.volume.aboutLabel',
              {
                defaultMessage: 'About {volume}',
              },
              values: {
                volume: ${new ByteSizeValue(volume * 1024 * 1024).toString()}
              },
            )} `,
```

Reply:
That's my concern initially, but I checked the languages we support (FR, CN, JP, via Google Translate), and they are all the same order.

But your proposed change makes more sense. Will update.

Reply:
We may add more languages in the future, that's not something we control--and you can't rely on google translate for this type of thing. Just follow the guide for creating translation strings: https://elastic.github.io/eui/#/utilities/i18n

Reply:
Done: https://github.com/elastic/kibana/pull/155250/commits/b53c012fdbb511222de089b60fdf20f99b7a54bd